### DISCUSSION TITLE
Open issuse

### DISCUSSION THREAD

**Discussion by LicinioFortunato** [stm32duino/Arduino_Core_STM32#2001] [Q&A] [answered]
hey there guys, i get two of this boards from a garage sells and try flash it but they iven get recognized in pc (linux or windows). any kind of help will apperciate. (they are new, never opened) 

(is exacly the same as this one

 https://pt.aliexpress.com/item/4000351115932.html?spm=a2g0o.detail.1000014.2.6ba53cd0PlLTdK&gps-id=pcDetailBottomMoreOtherSeller&scm=1007.40000.333918.0&scm_id=1007.40000.333918.0&scm-url=1007.40000.333918.0&pvid=74931d68-1395-4280-a15b-504bd4bbdbcc&_t=gps-id:pcDetailBottomMoreOtherSeller,scm-url:1007.40000.333918.0,pvid:74931d68-1395-4280-a15b-504bd4bbdbcc,tpp_buckets:668%232846%238112%23506&pdp_npi=3%40dis%21EUR%211.36%211.3%21%21%21%21%21%402103247416812522578152376efa86%2112000025684755250%21rec%21PT%21

 )

**Comment by fpistm** [comment, marked as answer]
Hi @LicinioFortunato 
First please avoid to open issue for this. Discussion or forum are better place for this.

It seems You can use a STLink to program it as we saw DIO, CLK and NRST pins available. Or you can probably flash over Serial, refer to https://www.st.com/resource/en/application_note/cd00167594-stm32-microcontroller-system-memory-boot-mode-stmicroelectronics.pdf

**Reply by LicinioFortunato** [reply]
Sorry. Next time I will open a discussion. I do not have st-link. Only UART, and USB converter (ch340)
